### PR TITLE
added better view for "No cars found" and Top Autoria cars added

### PR DESCRIPTION
**Pull request by o-rudych**
https://app.asana.com/0/1202525679663009/1202905421686449/f

image

### CODE REVIEW

**Review by oleksandr-danylchenko — CHANGES_REQUESTED** [review]
Can we see a screenshot?

**Comment by o-rudych** [comment]
@oleksandr-danylchenko, screenshot added

File: frontend/src/components/search-page/search-page.tsx [hunk 1]
```diff
@@ -21,7 +27,26 @@ const SearchPage: React.FC = () => {
...
+                  <h3 className={styles.noCarsTitle}>No cars found</h3>
+                </Container>
+                <Container className={styles.topAutoriaContainer}>
+                  <div>
+                    <div className={styles.topAutoriaHeader}>Top Autoria</div>
+                    <div className={styles.topAutoriaCards}>
+                      {topCars.map((car) => (
+                        <TopCarCard car={car} key={car.id} />
+                      ))}
+                    </div>
```
Comment (oleksandr-danylchenko) [inline frontend/src/components/search-page/search-page.tsx]: Let's create a component that wraps the top cars. So we can use it both here and on the landing

**Review by oleksandr-danylchenko — CHANGES_REQUESTED** [review]

**Comment by o-rudych** [comment]
@oleksandr-danylchenko, done.

**Review by oleksandr-danylchenko — APPROVED** [review]

**Pull request merged**